### PR TITLE
Implemented pidove_143 chirp

### PR DESCRIPTION
**Pull request by ufodynasty**

### CODE REVIEW

File: src/tcgwars/logic/impl/gen8/RebelClash.groovy [hunk 1]
```diff
@@ -3359,9 +3359,10 @@ public enum RebelClash implements LogicCardInfo {
         move "Chirp", {
           text "Search your deck for up to 2 Pokemon with a [F] Resistance, reveal them, and put them into your hand. Then, shuffle your deck."
           energyCost C
-          attackRequirement {}
+          attackRequirement {
+            assert my.deck : "There are no more cards in your deck."}
```
Comment (luongthomas) [inline src/tcgwars/logic/impl/gen8/RebelClash.groovy:3363]: Could you put the closing bracket on a new line?

File: src/tcgwars/logic/impl/gen8/RebelClash.groovy [hunk 2]
```diff
@@ -3359,9 +3359,10 @@ public enum RebelClash implements LogicCardInfo {
...
           onAttack {
-            // TODO
+            deck.search (max: 2 ,{it.cardTypes.is(POKEMON) &&  it.asPokemonCard().resistance.contains(F)}).showToOpponent("Chosen Pokémon cards.").moveTo(my.hand)
```
Comment (luongthomas) [inline src/tcgwars/logic/impl/gen8/RebelClash.groovy:3365]: What do you think of not using showToOpponent? The search action will still appear in the game log.  I've personally found the showToOpponent thing annoying but I'd like to hear your opinion

File: src/tcgwars/logic/impl/gen8/RebelClash.groovy [hunk 3]
```diff
@@ -3359,9 +3359,10 @@ public enum RebelClash implements LogicCardInfo {
         move "Chirp", {
           text "Search your deck for up to 2 Pokemon with a [F] Resistance, reveal them, and put them into your hand. Then, shuffle your deck."
           energyCost C
-          attackRequirement {}
+          attackRequirement {
+            assert my.deck : "There are no more cards in your deck."}
           onAttack {
-            // TODO
```
Comment (luongthomas) [inline src/tcgwars/logic/impl/gen8/RebelClash.groovy:3364]: Also don't forget to add a `shuffleDeck()`

File: src/tcgwars/logic/impl/gen8/RebelClash.groovy (hunk 2 above)
Comment (ufodynasty) [reply src/tcgwars/logic/impl/gen8/RebelClash.groovy:3366]: Personally I appreciate showToOpponent as it lets you read the cards found before play resumes. It lessens the burden on players to know exactly what cards do just from their name.

**Pull request merged**